tests/virt_autotest/guest_migration_base.pm

```diff
@@ -0,0 +1,93 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
Please specify only the year during which the file was changed, I guess 2016 only